admin/services/pin-generation.service.js

```diff
@@ -98,6 +97,15 @@ pinGenerationService.generateSchoolPassword = (school) => {
   return school
 }
 
+/**
+ * Generate Pupil Pin
+ * @returns {String}
+ */
+pinGenerationService.generatePupilPin = () => {
+  const chars = '23456789'
```

Comment:
If it is only numeric, why exclude the number '1'?

Reply:
@jon-shipley based on the AC they want to exclude number 1

Reply:
It makes sense to exclude the number 1 when mixed alphanumerics are being used to prevent ambiguous characters being interpreted incorrectly, but I see little benefit here.